### ISSUE TITLE
Warning: State value to set for "hm-rpc.2.INT0000002.1.CONTROL_MODE" has value "1" greater than max "-1"

### ISSUE THREAD

**Issue by cherberg** [ioBroker/ioBroker.hm-rpc#430]
Dear Developers, 

since an upgrade of the adapter mid of last year my iobroker Log is swarmed with these messages. Currently I am running latest stable version 1.15.0

It refers to my HM thermostats: 
https://www.eq-3.de/produkte/homematic/detail/homematic-funk-heizkoerperthermostat.html
which I am running in combination with wall thermostats:
https://www.eq-3.de/produkte/homematic/detail/homematic-funk-wandthermostat-aufputzmontage.html

The Control Mode is set to Manual = 1 and seems out of range of what the adapter is expecting.

There is no impact other than the warnings in the log but it is quite annoying and would be great if this could be fixed.

**Comment by foxriver76** [comment]
Have you deleted the affected objects once?

**Comment by cherberg** [comment]
> Have you deleted the affected objects once?
> 
> Else.. Dear User, it would be very helpful if you could provide more relevant information, like posting the structure of the affected raw object ;-)

No I have not done that, will the device be recreated automatically after I do this?

**Comment by foxriver76** [comment]
Yes it will. Delete - restart adapter and device will be there again

**Comment by cherberg** [comment]
Ah, great thanx. I did that and the issue seems solved. Thanx for your help, I will close this issue
